client/packages/common/src/intl/locales/en/common.json

```diff
@@ -71,6 +71,8 @@
   "heading.settings-display": "Display settings",
   "heading.settings-sync": "Synchronisation",
   "heading.shipments-to-be-picked": "Shipments to be picked",
+  "heading.details": "Details",
```

Comment:
Should this be in patient.json instead...? 🤔

Reply:
Seems generic enough to have it here. Or reuse the "label.details": "Details" from patient.json ?

Reply:
Maybe will leave it here if it's generic enough 🤔